estate_account/models/estate_property.py

```diff
@@ -0,0 +1,38 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details. 
+
+from odoo import fields, Command, models, _
```

Comment:
Imports must be in ASCII sorter manner.
```suggestion
from odoo import Command, _, fields, models
```